github-pr

```diff
@@ -12,11 +12,19 @@ def _check_required_fields(required, **args):
             sys.exit(1)
 
 
+def _return_status(commits):
+    commit_statuses = []
+    for commit in commits:
+        for commit_status in commit.get_statuses():
+            commit_statuses.append(commit_status.state)
+    return commit_statuses[-1]
```

Comment:
The last thing in the list is wrong. look up the most recent status for the most recent commit. Your loop here looks through all commits. `for commit in commits:`